tests/model_hub_tests/torch_tests/hf_transformers_models

```diff
@@ -104,7 +104,8 @@ google/mobilebert-uncased,mobilebert
 google/mobilenet_v1_0.75_192,mobilenet_v1
 google/mt5-base,mt5
 google/owlvit-base-patch32,owlvit
-google/pix2struct-docvqa-base,pix2struct,skip,Load problem
+google/pix2struct-docvqa-base,pix2struct,xfail,Provide example input
```

Comment:
I suspect there is a more descriptive error

Reply:
provided more informative error decription